docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/configuring-product-visibility-by-site-using-channels.md

```diff
@@ -1,6 +1,6 @@
-# Configuring Product Visibility by Site Using Channels
```

Comment:
I suggest removing the "by site" qualifier because even if a channel is not a liferay site (e.g. Amazon Marketplace), we will still make the product unavailable in the channel if it's filtered out. So this doc applies regardless of whether the channel is a site or not.